test/README.md

```diff
@@ -379,3 +379,8 @@ The realpath of the 'tmp' directory.
 * return [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
 
 Name of the temp directory used by tests.
+
+### WPT
+
+An implementation of parts of the W3C Web Platform Test harness. It is used in
+to test WHATWG URL conformance.
```

Comment:
A link would be helpful ([Web Platform Tests](https://github.com/w3c/web-platform-tests) and/or [testharness.js](https://github.com/w3c/testharness.js)). Also the second sentence seems to have an extra "in".